api_tests/src/swap_response.ts

```diff
@@ -59,7 +59,7 @@ export enum SwapStatus {
     /**
      * The communication was finalized and blockchain actions are needed or happening
      */
-    InProgress = "IN_PROGRESS",
+    InProgress = "IN_PROGRESS", // FIXME: Rename this to 'live' ... everywhere.
```

Comment:
I am not sure this is necessary. 
I am actually not sure `live` is less ambiguous than `in progress`.

Reply:
Good point about the states not being clearly defined, we should definitely define these clearly and have uniform terminology in the code base.

Reply:
According to the comments in the TS code this FIXME is correct
```
    /**
     * The swap was created but the communication phase hasn't yet been finalized
     */
    Created = "CREATED",
    /**
     * The communication was finalized and blockchain actions are needed or happening
     */
    InProgress = "IN_PROGRESS", // FIXME: Rename this to 'live' ... everywhere.
```

Reply:
Re-verted to using `InProgress` instead of `Live`.  Will add a ticket to rename this.

Reply:
Please note that IMHO, the state are clearly defined here.

I suggest that we indeed keep this up to date to harmonize the state terminology across the codebase and avoid using 3 different words to express the same state (live, in progress, finalized).